aggregator_mock/src/lib.rs

```diff
@@ -3,16 +3,16 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
 
-pub struct AggregatorResult<BigUint: BigUintApi> {
+pub struct AggregatorResult<M: ManagedTypeApi> {
     pub round_id: u32,
-    pub from_token_name: BoxedBytes,
-    pub to_token_name: BoxedBytes,
-    pub price: BigUint,
+    pub from_token_name: ManagedBuffer<M>,
+    pub to_token_name: ManagedBuffer<M>,
+    pub price: BigUint<M>,
     pub decimals: u8,
 }
 
-pub type AggregatorResultAsMultiResult<BigUint> =
-    MultiResult5<u32, BoxedBytes, BoxedBytes, BigUint, u8>;
+pub type AggregatorResultAsMultiResult<M> =
+    MultiResult5<u32, ManagedBuffer<M>, ManagedBuffer<M>, BigUint<M>, u8>;
 
 const DEFAULT_PRICE: u64 = 1_000u64;
 const DEFAULT_PRICE_DECIMALS: u8 = 2u8;
```

Comment:
No need to have the `u64` and `u8` at the end, you already wrote the type in front.